### ISSUE TITLE
`pdk update` is not idempotent for deletion of CI config files

### ISSUE THREAD

**Issue by AlmogCohen** [puppetlabs/pdk#593]
**Describe the bug**
`pdk update` re-execute delete changes although the files are already deleted. https://cl.ly/187b9cbca50f

**To Reproduce**
The simplest sequence of steps you have discovered which triggers the bug.
Specify this in your `.sync` file
```
appveyor.yml:
  delete: true
.gitlab-ci.yml:
  delete: true
```
Run `pdk update`, hit `Y`. Do this step twice and see the bug.

**Expected behavior**
No changes should be present after the first run and hitting `Y`

**Additional context**
- Your PDK installation method: brew on mac
- Your PDK version: 1.7.1
- Your operating system / platform - OSX

**Comment by ragnarkon** [comment]
I had the same issue in 1.7.1.

I believe it was fixed in 1.8.0 by #585

**Comment by rodjek** [comment]
Yep, that's right @ragnarkon, this was fixed and released in PDK 1.8.0.
